src/implementation/strided.jl

```diff
@@ -223,7 +223,7 @@ function blas_contract!(C, A, pA, conjA, B, pB, conjB, pAB, α, β, allocator)
         C_ = C
         _unsafe_blas_contract!(C_, A_, pA, B_, pB, ipAB, α, β)
     else
-        C_ = StridedView(TensorOperations.tensoralloc_add(TC, C, ipAB, false, true,
+        C_ = StridedView(TensorOperations.tensoralloc_add(TC, C, ipAB, false, Val(true),
```

Comment:
Is there a reason we explicitly use `TensorOperations` as a specifier here?

Reply:
No, not sure where this comes from.

Reply:
I think these definitions were at some point first in TensorKit.jl, and there was another implementation in TensorOperations, that I then replaced with the one from  TensorKit.